### ISSUE TITLE
ACCESSIBILITY AUDIT

### ISSUE THREAD

**Issue by lukas-stop** [lukas-stop/this-or-that-emotion-finder#13]
The overall end goal is to match [WCAG's AAA](https://www.w3.org/WAI/WCAG2AAA-Conformance) compliance rating.  At absolute minimum, this application should fit level A at all times to meet general accessibility standards.

Following this guide first for the initial audit: https://www.w3.org/TR/WCAG20/
Additional references will be used during this process.

Biggest concerns:
- All content should be properly organized, labeled, easy to navigate, and reachable using non-mouse movements (keyboard).
- Content should not be distracting or have rapid animations.  All content should have a contrast of at least 4.5:1 for normal text and 3:1 for large text.
- Text should be resizable
- All pages should be titled accordingly.  
- Help section should be clear and easy to understand.
- Error codes of any kind should be displayed to the user in a clear manner
- Name/Role/Value with every HTML element
